src/controllers/controller.bar.js

```diff
@@ -112,10 +112,9 @@ module.exports = function(Chart) {
 			var me = this;
 			var meta = me.getMeta();
 			var yScale = me.getScaleForId(meta.yAxisID);
-			var base = yScale.getBaseValue();
-			var original = base;
+			var base = 0;
```

Comment:
`getBasePixel()` will always clip to the screen, so if the `min` is 50 and the `max` is 100, it will return 50 for the base pixel even though the bar actually starts as 0. Before we had clipping implemented this was needed because otherwise the bar drew over the axis.